src/JCache-HZInstance.md

```diff
@@ -1,15 +1,14 @@
 ## JCache - Hazelcast Instance Integration
 
-You can retrieve `javax.cache.Cache` instances directly through `HazelcastInstance::getCache(String name)` method.
-The parameter `name` in `HazelcastInstance::getCache(String name)` is the full cache name except the Hazelcast prefix, i.e., `/hz/`. 
+You can retrieve `javax.cache.Cache` instances using the interface `ICacheManager` of `HazelcastInstance`. This interface has the method `getCache(String name)` where `name` is the prefixed cache name. The prefixes in the cache name are URI and classloader prefixes, which are optional. Please note that the method `getCache(String name)` does not use a cache name with Hazelcast prefix (`/hz/`). Please see the [`ICacheManager` Javadoc](http://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/core/ICacheManager.html) for details.
```

Comment:
Is the information about the `/hz/` prefix somehow important to users? Or is it rather an implementation detail and users will never meet this prefix?

Reply:
Talked with Vassilis, no. He suggested a useful content, I will add.